test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs

```diff
@@ -291,6 +291,20 @@ public RazorPagesTest(MvcTestFixture<RazorPagesWebSite.Startup> fixture)
             Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
         }
 
+        [Fact]
+        public async Task NullParameter_BadRequest()
```

Comment:
`PageHandlerCanReturnBadRequest` (the current name doesn't say what's being tested).